packages/simplebar/src/simplebar.js

```diff
@@ -449,6 +458,21 @@ export default class SimpleBar {
 
     this.hideNativeScrollbar();
 
+    // Set isOverflowing to false if scrollbar is not necessary (content is shorter than offset)
+    let offsetForXScrollbar = this.axis.x.isOverflowing
+      ? this.scrollbarWidth
+      : 0;
+    let offsetForYScrollbar = this.axis.y.isOverflowing
+      ? this.scrollbarWidth
+      : 0;
+
+    this.axis.x.isOverflowing =
+      this.contentEl.scrollWidth >
+      this.contentWrapperEl.offsetWidth - offsetForYScrollbar;
+    this.axis.y.isOverflowing =
+      this.contentEl.scrollHeight >
+      this.contentWrapperEl.offsetHeight - offsetForXScrollbar;
+
     this.axis.x.track.rect = this.axis.x.track.el.getBoundingClientRect();
     this.axis.y.track.rect = this.axis.y.track.el.getBoundingClientRect();
```

Comment:
Looks like these two lines are not needed now. `track.rect` is not used in this method and in other places it is recalculated.